ansible/inventories/devnet-9/group_vars/lighthouse.yaml

```diff
@@ -0,0 +1,55 @@
+# role: ethpandaops.general.bootstrap
+bootstrap_default_user_authorized_keys_github_team_cl:
+  - AgeManning
+  - ethdreamer
+  - paulhauner
+  - pawanjay176
+  - michaelsproul
+  - potuz
+  - nisdas
+  - antondlr
+  - realbigsean
+  - jimmygchen
+
+# role: validator_keys
+validator_keys_sync_files:
+  - src: "{{ inventory_dir }}/files/validator_keys/{{ inventory_hostname }}/keys/"
+    dest: "{{ lighthouse_validator_datadir }}/keys/"
+  - src: "{{ inventory_dir }}/files/validator_keys/{{ inventory_hostname }}/secrets/"
+    dest: "{{ lighthouse_validator_datadir }}/secrets/"
+
+validator_bls_sync_files:
+  - src: "{{ inventory_dir }}/files/validator_keys/{{ inventory_hostname }}/change_operations.json"
+    dest: "{{ lighthouse_validator_datadir }}/change_operations.json"
+    owner: lighthouse
+    group: lighthouse
+
+# role: ethpandaops.general.ethereum_node
+ethereum_node_cl: lighthouse
+
+# role: ethpandaops.general.lighthouse
+lighthouse_container_name: beacon
+lighthouse_container_image: "{{ default_ethereum_client_images.lighthouse }}"
+lighthouse_container_env:
+  VIRTUAL_HOST: "{{ ethereum_node_beacon_hostname }}"
+  VIRTUAL_PORT: "{{ ethereum_node_cl_ports_http_beacon | string }}"
+  LETSENCRYPT_HOST: "{{ ethereum_node_beacon_hostname }}"
+lighthouse_container_volumes:
+  - "{{ lighthouse_datadir }}:/data"
+  - "{{ lighthouse_auth_jwt_path }}:/execution-auth.jwt:ro"
+  - "{{ eth_testnet_config_dir }}:/network-config:ro"
+lighthouse_container_command_extra_args:
+  - --testnet-dir=/network-config
+  - --boot-nodes={{ ethereum_cl_bootnodes | join(',') }}
+  - --self-limiter=blob_sidecars_by_range:256/10
+  - --debug-level=debug
```

Comment:
lets switch to info everywhere? debug is a bit too spammy no? or would you prefer debug this time around in case we trigger issues inadvertantly?

Reply:
set a new default var for log level, and set the default to debug if its not defined. Should work nicely.